src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle implements Area, Draw {
```

Comment:
У всіх склас фігур в тебе дублюється поле String color. Для того щоб уникнути дублювання краще винести це поле в public abstract class Figure і після цього всі фігури екстендити від нього. Також там можна буде додати Figure implements Area, Draw
```suggestion
public class Circle extends Figure {
```